package.json

```diff
@@ -24,5 +24,9 @@
     "style-loader": "^3.3.1",
     "webpack": "^5.74.0",
     "webpack-cli": "^4.10.0"
+  },
+  "devDependencies": {
+    "stylelint": "^14.14.0",
+    "stylelint-config-standard": "^29.0.0"
   }
```

Comment:
- [ ] Please add in the scripts `"start"`

Reply:
Thanks for your review am done with the correction.